### ISSUE TITLE
Radon operator StringMatch is hard to use because field auto formats JSON

### ISSUE THREAD

**Issue by tmpolaczyk** [witnet/sheikah#1828]
The StringMatch operator has two arguments, the first one is a JSON map:

![Captura de pantalla de 2022-11-17 10-21-36](https://user-images.githubusercontent.com/44604217/202406797-1330bbe5-c327-4f6d-b2f4-caded8518caa.png)

It is easy to input a JSON map with one key:

```js
{"data":true}
```

But when trying to add another key, like

```js
{"data":true,"from":true}
```

It is not possible to write the comma after the first "true", because sheikah auto formats the JSON and removes the comma. The only way to enter the desired JSON is to use an external editor, and copy-paste the value.

So Sheikah should not remove that comma and let the user write freely.

**Comment by aesedepece** [comment]
Actually, for the sake of UX, matches should use a value => bool input component on the UI and form the JSON map from those. Value input should use an HTML input of the appropriate to the data type (which we know), and bools should be introduced through an HTML select box.

**Comment by tmpolaczyk** [comment]
> matches should use a value => bool input component on the UI and form the JSON map from those.

True, I always expected something like that, but for this issue I am fine with just being able to input the JSON manually.
